asv_bench/benchmarks/io/csv.py

```diff
@@ -285,6 +285,15 @@ def time_read_uint64_na_values(self):
         )
 
 
+class ReadUint8Integers(StringIORewind):
+    def setup(self):
+        arr = np.arange(10000).astype("uint8")
```

Comment:
This might cause a warning with numpy_dev, can you simply do something like

```
np.tile(np.arange(1, 250), n)
```

Reply:
Yes, thanks (see [24c3ef9](https://github.com/pandas-dev/pandas/pull/47168/commits/24c3ef9fb787d3bca4ccf888b18b812c47520d9c))